pallets/liquidity-pools/src/tests.rs

```
@@ -377,3 +377,43 @@ fn get_underlying_asset_id_by_wrapped_id_should_work() {
 		);
 	});
 }
+
+#[test]
+fn get_user_liquidation_attempts_should_work() {
+	ExtBuilder::default()
+		.pool_user_data_with_params(CurrencyId::DOT, ALICE, ONE_HUNDRED_DOLLARS, Rate::default(), true, 12)
+		.build()
+		.execute_with(|| {
+			assert_eq!(TestPools::get_user_liquidation_attempts(&ALICE, CurrencyId::DOT), 12);
+		});
+}
+
+#[test]
+fn set_user_liquidation_attempts_should_work() {
+	ExtBuilder::default()
+		.pool_user_data_with_params(CurrencyId::DOT, ALICE, ONE_HUNDRED_DOLLARS, Rate::default(), true, 0)
+		.build()
+		.execute_with(|| {
+			assert_ok!(TestPools::set_user_liquidation_attempts(&ALICE, CurrencyId::DOT, 15));
```

Attempt to set the parameter to 15 should raise an exception.

---

discussed*